src/inmanta/tracing.py

```diff
@@ -82,16 +137,16 @@ def span(
     *,
     _tags: Sequence[str] | None = None,
     _span_name: str | None = None,
-    _level: LevelName | None = None,
+    _level: LevelName | None | "logfire.LevelName" = None,
```

Comment:
This does not work, any suggestion?

Reply:
what doesn't work/ how does it nor work?